src/content/docs/references/docs/optionals_advanced.md

```diff
@@ -19,6 +19,73 @@ fn int! example() { ... }
 
 ## Handling `Empty` Optional
 
+### Return a default value when Optional is `Empty`
+If an expression returns a `Empty` Optional default assign using `??` operator.
```

Comment:
"The `??` operator allows us to return a default value if the result if the optional is `Empty`.